test/replication_test.go

```diff
@@ -45,7 +46,24 @@ func TestReplicationDatabaseInventory(t *testing.T) {
 		if err != nil {
 			t.Fatalf("Failed to open _system database: %s", describe(err))
 		}
-		inv, err := rep.DatabaseInventory(ctx, db)
+
+		version, err := c.Version(nil)
+		if err != nil {
+			t.Fatalf("Version failed: %s", describe(err))
+		}
+
+		ctx2 := ctx
+		if version.Version.CompareTo("3.2") >= 0 {
+			// RocksDB requires batchID
+			batch, err := rep.CreateBatch(ctx, db, 1337, time.Duration(60.0))
```

Comment:
Please add a comment explaining where the `1337` comes from.
Or change to:
```
serverID := 1337 // Random test value
batch, err := rep.CreateBatch(ctx, db, 1337, time.Second * 60)
```

Also not the change in time. I doubt 60ns is the intended value